packages/shared/src/components/widgets/ToggleWeekStart.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactElement } from 'react';
+import { useAuthContext } from '../../contexts/AuthContext';
+import { isNullOrUndefined } from '../../lib/func';
+import { DayOfWeek } from '../../lib/date';
+import { Radio, ClassName as RadioClassName } from '../fields/Radio';
+import useProfileForm from '../../hooks/useProfileForm';
+
+const getDefaultStartOfWeek = (weekStart?: number): string => {
```

Comment:
Probably not used elsewhere now, but feels like a generic function really?

Reply:
EG shouldn't we place it in [packages/shared/src/lib/date.ts](https://github.com/dailydotdev/apps/pull/3386/files#diff-1a08dac1b61d1899f836dbf8a53e5757ae64f00fbac8b7e56af7b9933456d44c)

Reply:
I created that before I created `date.ts` 🙃 
But I've moved it now.